parts/k8s/kubernetesmasterresourcesvmss.t

```diff
@@ -24,13 +24,20 @@
     "name": "[variables('clusterKeyVaultName')]",
     "apiVersion": "[variables('apiVersionKeyVault')]",
     "location": "[variables('location')]",
-    {{ if UseManagedIdentity}}
+    {{if UseManagedIdentity}}
+    {{if UserAssignedIDEnabled}}
+    "dependsOn": [
+     "[concat('Microsoft.Compute/virtualMachines/', variables('masterVMNamePrefix'), copyIndex())]",
```

Comment:
I don't think you can do `copyIndex()` for VMSS.

Reply:
Thanks Rita, will fix this in the upcoming iteration.

Reply:
Oh, this is still not fixed in the current iteration :-(

Reply:
This code has moved of this context.